libraries/AP_Common/AP_Common.h

```diff
@@ -144,15 +144,6 @@ struct PACKED Location {
     int32_t lng;                                        ///< param 4 - Longitude * 10**7
 };
 
-/*
-  home states. Used to record if user has overridden home position.
-*/
-enum HomeState {
```

Comment:
Great to remove this, but should be in its own commit instead of inside an *AP_AHRS* commit.

Reply:
Will fix.